Feliz/Html.fs

```diff
@@ -309,8 +309,8 @@ type Html =
     static member inline form xs = Interop.createElement "form" xs
     static member inline form (children: #seq<ReactElement>) = Interop.reactElementWithChildren "form" children
 
-    [<Obsolete("Html.fragment is obsolete, use React.fragment instead. This function will be removed in the coming v1.0 release")>]
-    static member inline fragment xs = Fable.React.Helpers.fragment [] xs
+    // [<Obsolete("Html.fragment is obsolete, use React.fragment instead. This function will be removed in the coming v1.0 release")>]
+    // static member inline fragment xs = Fable.React.Helpers.fragment [] xs
```

Comment:
The comment says the function would be removed in v1.0. As the current version is already far ahead of that I just removed instead of trying to update it.